packages/oruga/src/components/steps/tests/StepsExample.vue

```diff
@@ -0,0 +1,127 @@
+<script setup lang="ts" generic="T extends Record<string, unknown> | string">
+import { type PropType, ref } from "vue";
+import OSteps from "@/components/steps/Steps.vue";
+import OStepItem from "@/components/steps/StepItem.vue";
+import OButton from "@/components/button/Button.vue";
+
+defineProps({
+    isVertical: {
+        type: Boolean,
+        default: false,
+    },
+    isAnimated: {
+        type: Boolean,
+        default: false,
+    },
+    isRounded: {
+        type: Boolean,
+        default: false,
+    },
+    hasNavigation: {
+        type: Boolean,
+        default: false,
+    },
+    labelPosition: {
+        type: String as PropType<"bottom" | "left" | "right">,
+        default: "bottom",
+    },
+    isStepsClickable: {
+        type: Boolean,
+        default: false,
+    },
+    isProfileSuccess: {
+        type: Boolean,
+        default: false,
+    },
+    showSocial: {
+        type: Boolean,
+        default: false,
+    },
+    customNavigation: {
+        type: Boolean,
+        default: false,
+    },
+});
+
+const activeStep = ref<number>(1);
+const enableProfileActivateEvent = ref(false);
+
+const onProfileActivate = () => {
+    if (enableProfileActivateEvent.value) {
+        alert("Profile Activated");
+    }
+};
+</script>
+
+<template>
+    <!-- Taken from the first example in the documentation -->
+    <o-steps
+        v-model="activeStep"
+        :vertical="isVertical"
+        :animated="isAnimated"
+        :rounded="isRounded"
```

Comment:
You only need to specify props for this test component that you will not be using only for the root element. Anything else can be passed to OSteps with `:v-bing=$attrs`.

Reply:
@mlmoravek I tried taking these out but realized: I want to toggle one prop at a time to better isolate a11y problems. In order to do that, I want to set default values in props that stay consistent regardless of what the component's defaults are. What do you think about leaving this as is?

Reply:
I think for now its fine :) I can refactor it late on when we will add more steps tests. 
To demonstrate how we can benefit from axe tests its good.